XRTK-Core/Packages/com.xrtk.core/EventDatum/Input/GraphicInputEventData.cs

```diff
@@ -43,7 +43,6 @@ public void Clear()
             pressPosition = Vector2.zero;
             rawPointerPress = null;
             scrollDelta = Vector2.zero;
-            selectedObject = null;
```

Comment:
This was removed so that the event system wouldn't override our currently selected object when scrolling.